### PR TITLE
Fix error message for lsp server.

### PR DESCRIPTION
**Pull request by BugDiver**
Fixes https://github.com/getgauge/gauge-vscode/issues/439

### CODE REVIEW

File: api/lang/runner.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ var lRunner langRunner
 func startRunner() error {
 	var err = connectToRunner()
 	if err != nil {
-		return fmt.Errorf("unable to connect to runner : %s", err.Error())
+		return fmt.Errorf("Gauge could not connect to the runner. Check the logs for more info")
```
Comment (zabil) [inline api/lang/runner.go:34]:
```suggestion
		return fmt.Errorf("Gauge could not run the tests. Check the logs for more info")
```

File: api/lang/server.go [hunk 2]
```diff
@@ -278,8 +274,9 @@ func startLsp(logLevel string) (context.Context, *jsonrpc2.Conn) {
 func initializeRunner() error {
 	id, err := getLanguageIdentifier()
 	if err != nil || id == "" {
-		logDebug(nil, "Current runner is not compatible with gauge LSP.")
-		return err
+		e := fmt.Errorf("Current runner is not compatible with gauge LSP. Some of the gauge lsp feature will not work as expected.")
```
Comment (zabil) [inline api/lang/server.go:277]:
```suggestion
		e := fmt.Errorf("There are version incompatibilities between Gauge and it's plugins in this project. Some features will not work as expected.")
```

File: api/lang/runner.go (hunk 1 above)
Comment (BugDiver) [reply api/lang/runner.go:34]: This error is not meant for Execution. It's while starting the LSP server. Not sure if it should say something about Execution, could be confusing.

Comment (zabil) [reply api/lang/runner.go:34]: `the runner` is a bit confusing. Can this be
`Gauge could not initialise. Check the logs for more info`?

Comment (BugDiver) [reply api/lang/runner.go:34]: 👍

**Pull request merged**